WowPacketParser/Store/Objects/UpdateFields/LegacyImplementation/UnitData.cs

```diff
@@ -66,6 +66,14 @@ public uint[] NpcFlags
 
         public int FactionTemplate => UpdateFields.GetValue<UnitField, int>(UnitField.UNIT_FIELD_FACTIONTEMPLATE);
 
+        public long Health => UpdateFields.GetValue<UnitField, int>(UnitField.UNIT_FIELD_HEALTH);
+
+        public long MaxHealth => UpdateFields.GetValue<UnitField, int>(UnitField.UNIT_FIELD_MAXHEALTH);
+
+        public int Mana => UpdateFields.GetValue<UnitField, int>(UnitField.UNIT_FIELD_POWER);
```

Comment:
Classic is based on modern client and has all of its power handling where type is not tied to a specific slot but must be retrieved with the help of ChrClassesXPowerTypes.db2 (mana always ends in slot 0) so it would be better to just expose Power/MaxPower as arrays

Reply:
Perhaps. But the rest of the information is useless. There is no custom handling of other powers in classic. This is mostly a simplification of the situation based on what we use in vanilla through wotlk.

Reply:
Im pushing for this because updatefield structure files like `UpdateFields/V8_2_0_30898/UnitData.cs` are autogenerated so its better to not add additional fields to its base interface

Reply:
I can't comment on bfa stuff, but I just want to add that mana is exported correctly for classic with the current method.

Reply:
He meant that the interface should be mindful of best practices for the whole software package.